archinstall/lib/interactions/disk_conf.py

```diff
@@ -213,7 +213,7 @@ def _boot_partition(sector_size: disk.SectorSize, using_gpt: bool) -> disk.Parti
 		flags.append(disk.PartitionFlag.ESP)
 	else:
 		start = disk.Size(3, disk.Unit.MiB, sector_size)
-		size = disk.Size(203, disk.Unit.MiB, sector_size)
+		size = disk.Size(1, disk.Unit.GiB, sector_size)
```

Comment:
As this is equivalent to the GPT version now, can we move this above the if-else

Reply:
Done.